src/extension/completions/vscode-node/completionsCoreContribution.ts.txt

```diff
@@ -30,6 +34,11 @@ export class CompletionsCoreContribution extends Disposable {
 			const unificationStateValue = unificationState.read(reader);
 			const configEnabled = configurationService.getExperimentBasedConfigObservable<boolean>(ConfigKey.Internal.InlineEditsEnableGhCompletionsProvider, experimentationService).read(reader);
 
+			if (this._copilotToken.read(reader)?.isNoAuthUser) {
+				// TODO@bpasero revisit this in the future
```

Comment:
Might be good to open an issue for this, assign it to some future milestone, and reference the issue here so as to not lose track.